src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -112,7 +112,7 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
-            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (self::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
```

Comment:
I think that would imply isolating it in a public method. Or maybe… in a private method, and then do a bit of black magic to make it public with reflection. I'll see what I can do.